libraries/abstractions/secure_sockets/include/iot_secure_sockets.h

```diff
@@ -221,7 +221,7 @@ typedef struct SocketsSockaddr
  * * `pdPASS` if everything succeeds
  * * `pdFAIL` otherwise.
  */
-lib_initDECLARE_LIB_INIT( SOCKETS_Init );
```

Comment:
I am OK with this I am pretty sure lib_initDECLARE_LIB_INIT is not used anymore.